website/static/js/comment.js

```diff
@@ -326,11 +327,12 @@ var CommentModel = function(data, $parent, $root) {
         self.author = self.$root.author;
     }
 
-    self.contentDisplay = ko.observable(markdown.full.render(self.content()));
+    var linkify_opts = { target: function (href, type) { return type === 'url' ? '_top' : null; } };
```

Comment:
As per COSDev, use lowerCamelCase for variable names. I'll fix this when I merge.